release/models/system/openconfig-system-controlplane.yang

```diff
@@ -45,10 +45,17 @@ module openconfig-system-controlplane {
        The specified control-plane ACL is applied to traffic received by the
        control-plane of the system.";
 
-    oc-ext:openconfig-version "0.2.0";
+    oc-ext:openconfig-version "0.3.0";
     oc-ext:catalog-organization "openconfig";
     oc-ext:origin "openconfig";
 
+    revision "2023-10-16" {
+      description
+        "Add embaded-limits container for  vendor-neutral, abstract expression
```

Comment:
Spelling correction for embedded, here and throughput the model file.  Note, you may want to consider using the term 'built-in' as this is found elsewhere on OC models to refer to functionality that is defined by the target device or system. [Example in QoS](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-qos.html#qos-classifiers-classifier-terms-term-conditions-transport-config-builtin-detail),   [Example in Policy Forwarding](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-policy-forwarding-policies-policy-rules-rule-transport-config-builtin-detail), [Example in ACL](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-acl.html#acl-acl-sets-acl-set-acl-entries-acl-entry-transport-config-builtin-detail)

```suggestion
        "Add embedded-limits container for  vendor-neutral, abstract expression
```